doc/source/skips/1-values.rst

```diff
@@ -0,0 +1,168 @@
+======================================
+SKIP-1: scikit-image mission statement
+======================================
+
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Status: Active
+:Type: Process
+:Created: 2018-12-08
+:Resolved:
+:Resolution:
+:Version effective: 0.16
+
+Abstract
+--------
+
+scikit-image should adopt the document below as its mission statement. This
+statement will feature prominently in the scikit-image home page and readme,
+as well as the contributor and core developer guides. Decisions about the API
+and the future of the library would be referenced against this document. (See
+:ref:`governance`.)
+
+In July 2018, I published a blog post that broadly outlined what I would want
+from a roadmap for scikit-image [1]_, but requested comments from the community
+before it would be finalized. I consider that we have collected comments for
+long enough and can move forward with formal adoption. Most comments were
+positive, so below I'll just summarize the “negative” comments under “rejected
+ideas”.
+
+Detailed description
+--------------------
+
+(Or: What problem does this proposal solve?)
+
+Over the past few years, scikit-image has been slightly “adrift”, with new and
```

Comment:
I think adrift here is a strange statement: our model was never one where we had in mind to run hard in any direction at the cost of another. I also don't think anything proposed in this document will change the volunteer nature of the project, and the availability on resources to work on this or that.

If we decided to have a more goal-driven approach, that'd be a different matter.  You can think of @alexandrejaguar's postdoc, e.g., as one attempt to focus on a specific direction of development.